pypdns/__init__.py,sha256=cJgK8t-FF9PZfOd_tXbXQpYXus07V93q22WwEErH5tc,1069
pypdns/api.py,sha256=avy7pbvp075Tb3-Qd8PXxU4t84OIlJQ5jpbOj4ygTNk,18899
pypdns/errors.py,sha256=z1MDSVuN0I553AczIVQhJA4U7JPoChkRAv1dOFaNLPw,474
pypdns/py.typed,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pypdns-2.2.1.dist-info/LICENSE,sha256=o1p6jKkqoo0k2KXZ23F8zTPaRGIoF8Gu9ADxi-QiUyo,1576
pypdns-2.2.1.dist-info/METADATA,sha256=1Orz4eg-EMbp-Md0q-0PH2XpasEREFwRa7LolwdMDro,2758
pypdns-2.2.1.dist-info/WHEEL,sha256=FMvqSimYX_P7y0a7UY-_Mc83r5zkBZsCYPm7Lr0Bsq4,88
pypdns-2.2.1.dist-info/entry_points.txt,sha256=9MC-9q1FbTVA0TQgOgc4JMUz9comV4RsGdeDel5gdxg,36
pypdns-2.2.1.dist-info/RECORD,,
